mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/PdfRoutes.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.mulighetsrommet.api.routes.v1
+
+import com.lowagie.text.Document
+import com.lowagie.text.Font
+import com.lowagie.text.FontFactory
+import com.lowagie.text.Paragraph
+import com.lowagie.text.pdf.PdfWriter
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.serialization.Serializable
+import org.apache.commons.io.output.ByteArrayOutputStream
+
+fun Route.pdfRoutes() {
+    post("/pdf") {
+        val dbo = call.receive<PdfKvittering>()
+        val pdfBytes = generatePdfInMemory(dbo)
+
+        call.response.headers.append("Content-Disposition", "attachment; filename=\"kvittering.pdf\"")
+        call.respondBytes(pdfBytes, contentType = io.ktor.http.ContentType.Application.Pdf)
+    }
+}
+
+@Serializable
```

Comment:
Burde det flyttes til domain/dto?

Reply:
Jeg er egentlig fan av at disse dataklassene ligger tettere der de blir brukt, men om vi tror de skal brukes flere steder i kodebasen så bør vi nok flytte dem. Kan diskutere litt på mandag hvordan vi ser for oss bruken av pdf-genereringen.